macantine/etl.py

```diff
@@ -386,25 +383,56 @@ def _load_data_parquet(self, filename):
             self.df.to_parquet(parquet_file)
 
     def _load_data_xlsx(self, filename):
-        with default_storage.open(filename + ".xlsx", "wb") as excel_file:
-            df_export = self.df.copy()
-            df_export = datetimes_to_str(df_export)
-            df_export.to_excel(excel_file, index=False)
+        try:
+            chunk_size = 1000
+            start_row = 0
+
+            # Convert datetime to string
+            df = datetimes_to_str(self.df.copy())  # Assuming it converts datetimes to strings
+
+            # Create an in-memory bytes buffer
```

Comment:
est-ce qu'on pourrait avoir des pbs si on mets tout un gros fichier `in-memory` ? Est-ce que c'est possible d'ecrire le fichier progressivement, avec le mode "a" ? Après, si tu fais ça on aurait p-e besoin de gérer le cas ou un chunk donne une erreur... on pourrait supprimer les contenus du fichier

Reply:
Je propose b65d3fba6bfc2ac5dd694dc911b9c5fc78602488
Je déplace juste la transfo au sein de la boucle pour faire la transfo sur le chunk et pas du dataframe entier.

Cela permet d'avoir un seul dataframe en mémoire et non deux. Le problème du `in-memory` est alors moins important et partagé par tout le reste du code des ETL.

Reply:
J'aime le refacto.

Tu introduis tjs `output` dans cette PR - c'est ça dont j'ai un peu peur. Est-ce que tu sais si il y a une limite sur la taille de ça ?

Reply:
La seule limite est la taille de la RAM. 
Notre fichier fait actuellement 12MB en csv (le type de fichier le plus lourd).
A vue d'oeil, il ne fera jamais plus de 30MB. On est donc assez large sur cette contrainte !